pkgs/development/python-modules/appnope/default.nix

```diff
@@ -1,21 +1,27 @@
 { lib
 , buildPythonPackage
-, fetchPypi
+, fetchFromGitHub
+, pytestCheckHook
 }:
 
 buildPythonPackage rec {
   pname = "appnope";
   version = "0.1.3";
 
-  src = fetchPypi {
-    inherit pname version;
-    hash = "sha256-Ar2RxN6Gn7seHFCq/ECYgnp6VKsvOdncumyVR+2SDiQ=";
+  src = fetchFromGitHub {
+    owner = "minrk";
+    repo = "appnope";
+    rev = version;
+    hash = "sha256-JYzNOPD1ofOrtZK5TTKxbF1ausmczsltR7F1Vwss8Sw=";
   };
 
+  checkInputs = [
+    pytestCheckHook
+  ];
+
   meta = {
     description = "Disable App Nap on macOS";
-    homepage    = "https://pypi.python.org/pypi/appnope";
-    platforms   = lib.platforms.darwin;
```

Comment:
The upstream package metadata says otherwise https://github.com/minrk/appnope/blob/main/setup.py#L51

Reply:
> The upstream package metadata says otherwise

The upstream code also explicitly handles non-Darwin, and this fallback works fine on Linux: https://github.com/minrk/appnope/blob/38e1e1252c263e4f110b427fd13cb81ac6fe56cb/appnope/__init__.py#L10-L13

:shrug:

> Does corrscope not handle this correct?

Since the package itself doesn't error on Linux in a "not supported here" kind of way, I don't see a clear reason why corrscope should have to care.

> Also this needs a comment otherwise the next person will put that line right back.

Can do.

Reply:
well fine for me, if you add a short comment and yourself as a maintainer

Reply:
Not sure if I'm the best person to maintain this Python module, or if "to fix this, you must sign up to maintain it" is the best approach, but if that's what it takes to get this merged then *sure*…